layers/meta-resin-raspberrypi/recipes-bcm/bootfiles/rpi-config_git.bbappend

```diff
@@ -3,6 +3,12 @@ do_deploy_append() {
     echo "dtparam=i2c_arm=on" >>${DEPLOYDIR}/bcm2835-bootfiles/config.txt
     # Enable SPI by default
     echo "dtparam=spi=on" >>${DEPLOYDIR}/bcm2835-bootfiles/config.txt
+    # Disable firmware splash by default
+    echo "disable_splash=1" >>${DEPLOYDIR}/bcm2835-bootfiles/config.txt
+    # Disable firmware warnings showing in non-debug images
+    if [ -z "${@bb.utils.contains('DISTRO_FEATURES','debug-image','1','',d)}"]; then
```

Comment:
This line is really confusing. I think it would be why clearer:

`if ! ${@bb.utils.contains('DISTRO_FEATURES','debug-image','true','false',d)}; then`